cmd/aro/mirror.go

```diff
@@ -46,6 +46,16 @@ func mirror(ctx context.Context, log *logrus.Entry) error {
 		return err
 	}
 
+	mirrorToInt := false
+	var intdstauth *types.DockerAuthConfig
+	if _, found := os.LookupEnv("INT_DST_AUTH"); found {
```

Comment:
Instead of this, I suggest taking some input which is the name of the repo to sync to.  Either an env var or a command line argument.  Let's sync the quay content to the int repo too.  Although it's a bit heavy, it moves us towards enabling the ACR token feature in INT.